flake.nix

```diff
@@ -0,0 +1,10 @@
+{ inputs.utils.url = "github:numtide/flake-utils";
+
+  outputs = { utils, ... }:
+    utils.lib.eachDefaultSystem
+      (system:
+         { packages =
+             (import ./.  { pkgs = import ./pinned.nix { inherit system; }; }).inputs;
```

Comment:
If this project were to move to flakes, it would be better to use the Flake's `inputs` and not even reference the `./pinned.nix` file. Instead a `default.nix` file could be derived through [`flake-compat`](https://github.com/edolstra/flake-compat) until it is decided that a flake is all that is needed and legacy support isn't.